src/test/python/ovms_py_tensor_test.cpp

```diff
@@ -0,0 +1,120 @@
+//*****************************************************************************
+// Copyright 2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <utility>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "../../ovms_py_tensor.hpp"
+
+#include <pybind11/pybind11.h>
+
+using namespace ovms;
+
+namespace py = pybind11;
+
+class OvmsPyTensorTest : public OvmsPyTensor {
```

Comment:
Why are you inherit from OvmsPyTensor if those fields are already accessible (public?)